templates/app/models/sequelize-model.hbs

```diff
@@ -20,20 +20,43 @@ module.exports = (sequelize, DataTypes) => {
     schema: process.env.DATABASE_SCHEMA,{{/if}}
   });
 {{#if noId}}  {{modelName}}.removeAttribute('id');{{/if}}
-  // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
+  // This section contains the relationships for this model.
+  // See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
```

Comment:
This was specified in another PR why do you change this line ?

Reply:
I'll fix this.

Reply:
Fixed by Arnaud

-